### DISCUSSION TITLE
Multimaterial paint issue...

### DISCUSSION THREAD

**Discussion by 3Dgraf64** [supermerill/SuperSlicer#3757] [General]
Hi Supermerill!
Thanks for your great work on this project.  Answering all these posts is a huge job.
Question:  I'm new to Superslicer.  I use it for my ERCF on my Voron 2.4.  Trying to paint this object: 
![Untitled2](https://user-images.githubusercontent.com/92392776/228053219-5e627424-be4f-4163-af96-7e9a6429d46e.png)
![Untitled1](https://user-images.githubusercontent.com/92392776/228051295-f68f7a50-a2b9-4929-87b1-bb876b0c847e.png)
Painting itself is easy but the slicing result is not great.  What is causing these yellow blobs in my painted red parts?  I've looked for the answer on the web without success.
Thanks!
Edit:  This does not happen in Prusaslicer...
![Untitled3](https://user-images.githubusercontent.com/92392776/229914990-28cc4640-04df-49b7-8ca2-e3409473d111.png)

**Comment by lokertr** [comment]
Anyone have any ideas on this?

**Reply by supermerill** [reply]
Can't do anything without a project that help me to reproduce.

**Comment by 3Dgraf64** [comment]
Problem solved.  Thanks!

**Reply by lokertr** [reply]
What was the solution?

**Reply by 3Dgraf64** [reply]
Got it on the Voron ERCF Facebook group:
"[Richard Wood]
Set one of the resolutions to 0 in slicing. Not sure which one but can look in the morning. It does strange things with the perimeters. Setting the (not sure if it's internal resolution or the other one) to 0 fixed it. Only an issue with multi colour"

**Reply by supermerill** [reply]
still, can I get a little project file that replicate the behavior? To have it fixed properly.

**Reply by lokertr** [reply]
Yep slicing resolution set to 0 fixes it. I am uploading a project file right now that is in its original broken state. You can see the problem after slicing the file and looking at gcode preview, the problem will be readily apparent in the painted eyes. You can rectify the issue in the project file by changing slice resolution to 0. 
[SV04Love_Dragon.zip](https://github.com/supermerill/SuperSlicer/files/14125010/SV04Love_Dragon.zip)

**Reply by 3Dgraf64** [reply]
@supermerill   Here it is:
[PikachuThick_tail.zip](https://github.com/supermerill/SuperSlicer/files/14129685/PikachuThick_tail.zip)

**Reply by supermerill** [reply]
should be fixed in next nightly
